stdlib/public/core/FloatingPoint.swift

```diff
@@ -328,7 +328,14 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable
   /// `infinity` is greater than this value.
   static var greatestFiniteMagnitude: Self { get }
 
-  /// The mathematical constant pi.
+  /// The mathematical constant π, approximately equal to 3.14159.
+  ///
+  /// This constant is most often defined as the numerical value of the ratio
+  /// of a circle's circumference to its diameter, which is the same for all
+  /// circles.
+  ///
+  /// A common application of this constant in geometry is to measure a half
+  /// revolution in radians. Used like this, it is equivalent to 180 degrees.
   ///
   /// This value should be rounded toward zero to keep user computations with
```

Comment:
Instead of "should be", this part of the comment should say "is".

Reply:
I interpreted that as being guidance to the interface implementation. Happy to change the original though if that's not the case.

Reply:
It was when I wrote it, but that's not really the function of this sort of documentation any longer.